src/components/breadcrumb/EllipsisCrumb.js

```diff
@@ -7,13 +7,17 @@ import PlainButton from '../plain-button';
 
 type Props = {
     children: React.Node,
+    menuButton?: React.Node,
 };
 
-const EllipsisCrumb = ({ children }: Props) => (
-    <DropdownMenu>
-        <PlainButton className="breadcrumb-toggler">⋯</PlainButton>
-        <Menu>{children}</Menu>
-    </DropdownMenu>
-);
+const EllipsisCrumb = ({ children, menuButton }: Props) => {
```

Comment:
This used to be an icon, but now is a button to consider custom styling and restrict one specific component rather than multiple